styles/main.css

```diff
@@ -0,0 +1,764 @@
+@import url('https://fonts.googleapis.com/css2?family=Oxygen:wght@300;400;700&display=swap');
+@font-face {
+    font-family: 'et-line';
+    src: url('../resources/et-line-font/fonts/et-line.eot');
+    src: url('../resources/et-line-font/fonts/et-line.eot?#iefix') format('embedded-opentype'), url('../resources/et-line-font/fonts/et-line.woff') format('woff'), url('../resources/et-line-font/fonts/et-line.ttf') format('truetype'), url('../resources/et-line-font/fonts/et-line.svg#et-line') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+* {
+    box-sizing: border-box;
+    font-family: "et-line", "Oxygen", sans-serif;
+    --my-grey: #f9f9f9;
+    --my-dark-grey: #ececec;
+    --my-green: rgb(140, 201, 187);
+    --font-main-color: #4f4f4f;
+    --font-secundary-color: #858585;
+    --font-title-color: #323232;
+    scroll-behavior: smooth;
+}
+
+body {
+    margin: 0px;
+    display: flex;
+    flex-direction: column;
+}
+
+p {
+    font-size: 0.75rem;
+    line-height: 1.8;
+    color: var(--font-main-color);
+}
+
+h1 {
+    color: var(--font-title-color);
+}
+
+h3 {
+    color: var(--font-title-color);
+    font-size: 1rem;
+}
+
+.icon {
+    font-size: 1.5rem;
+}
+
+.icon-rounded {
+    background-color: black;
+    border-radius: 50%;
+}
+
+.icon-behance {
+    color: white;
+    background-color: var(--font-title-color);
+    border-radius: 50%;
+}
+
+header {
+    order: 1;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0%;
+    background-color: white;
+    box-shadow: 0px 1px 5px rgb(240, 240, 240);
+    z-index: 50;
```

Comment:
I recommend using indexes between 1 and 10, if you need a higher number it might be useful to check the position of the parent elements and its z index